lib/runctl.js

```diff
@@ -155,6 +172,29 @@ function onRequest(req, callback) {
   }
 }
 
+function requestAllTargets(req, callback) {
```

Comment:
Well, for strong-pm, I think what it does now is kill the supervisor master, taking down all the workers, then restart.

By telling the supervisor master the env is changing, that allows it to change its own env, then do a zero-downtime cluster restart.

Anyhow, I like that supervisor moves towards just implementing the primitive mechanism, which is what you have, and then pm can compose those mechanisms as it sees fit. So, I like the feature, and I would NOT add a console message, but I would add a note in the usage: "not all apps notice when their process environment changes, you may need to restart your app for env changes to take effect."